test/specs/upload/tests/03.update_w_upload.js

```diff
@@ -0,0 +1,180 @@
+var uploadUtils = require("../utils.js");
+var utils = require("../../../utils/utilities.js");
+
+exports.execute = function (options) {
+    var exec = require('child_process').execSync;
+    var FileAPI = require('file-api'), File = FileAPI.File;
+    File.prototype.jsdom = true;
+
+    describe("updating reference objects, ", function () {
+        var catalogId = process.env.DEFAULT_CATALOG,
+            schemaName = "upload";
+
+        describe("for updating asset columns, ", function() {
+            var reference, file1_column, file2_column,
+                file1_columnName = "file1_uri",
+                file2_columnName = "file2_uri",
+                tableName = "file_update_table",
+                sortBy = "key",
+                baseUri = options.url + "/catalog/" + catalogId + "/entity/" + schemaName + ':' + tableName + "/key=1";
+
+            // For update data
+            var file1_url, file1_validRow, file2_url, file2_validRow;
+
+            var files = [{
+                    name: "testfile500kb.png",
+                    size: 512000,
+                    displaySize: "500KB",
+                    type: "image/png",
+                    hash: "4b178700e5f3b15ce799f2c6c1465741",
+                    hash_64: "SxeHAOXzsVznmfLGwUZXQQ=="
+                }, {
+                    name: "testfile5MB.txt",
+                    size: 5242880,
+                    displaySize: "5MB",
+                    type: "text/plain",
+                    hash: "08b46181d7094b5ece88bb389c7499af",
+                    hash_64: "CLRhgdcJS17OiLs4nHSZrw=="
+                }
+            ];
+
+            beforeAll(function (done) {
+
+                for (var i=0; i<files.length; i++) {
+                    var file = files[i];
+                    var filePath = "test/specs/reference/files/" + file.name;
```

Comment:
Change the path to `test/specs/upload/files`. Then you can remove `test/specs/reference/files` folder since we don't have any other file upload test cases in reference.